config/networks.json

```diff
@@ -258,7 +470,14 @@
     "nativeCurrency": "OKB",
     "wrappedNativeAddress": "0xe538905cf8410324e03a5a23c1c177a474d59b2b",
     "status": "active",
-    "type": "mainnet"
+    "type": "mainnet",
+    "rpcUrl": "https://rpc.xlayer.tech",
+    "explorerType": "etherscan",
```

Comment:
just thinking....would it maybe make sense to rename this to verificationType instead of explorerType? 
I am not sure....